sat/handler.go

```diff
@@ -22,11 +32,8 @@ func (s *Sat) handler(exec *executor.Executor) vk.HandlerFunc {
 
 		result, err := exec.Do(s.j, req, ctx, nil)
 		if err != nil {
-			// check if the error type is rt.RunErr, because those are handled differently
-			if returnedErr, isRunErr := err.(rt.RunErr); isRunErr {
-				runErr = returnedErr
```

Comment:
I don't think we can lose this line, otherwise `runErr` will never get set

Reply:
I reworked the logic such that every eventuality gets hit, and the branching is a bit cleaner

Reply:
Nice! I like that.